span.go

```diff
@@ -58,3 +62,30 @@ type Span interface {
 	// TraceContext.
 	AddToGoContext(goCtx context.Context) (Span, context.Context)
 }
+
+// See Span.Log(). Every LogData instance should specify at least one of Event
+// and/or Payload.
+type LogData struct {
+	// The timestamp of the log record; if set to the default value (the unix
+	// epoch), implementations should use time.Now() implicitly.
+	Timestamp time.Time
+
+	// Event (if non-empty) should be the stable name of some notable moment in
```

Comment:
empty string is the default "zero" value for a string field in Go, so it's fine to treat it as "not set".